build woes on master

I am unable to build master. I get errors of this type, referring to various dependencies:

```
make: *** No rule to make target 'deps/vcflib/include/*', needed by 'lib/libvcflib.a'.  Stop.
```

There is no deps/vcflib/include/, so the target isn't appropriate.

--- a/Makefile
+++ b/Makefile
@@ -145,7 +145,7 @@ $(LIB_DIR)/libhts.a:
 $(LIB_DIR)/libxg.a: $(LIB_DIR)/libsdsl.a $(LIB_DIR)/libprotobuf.a $(CPP_DIR)/vg.pb.o $(INC_DIR)/dynamic.hpp $(XG_DIR)/src/xg.cpp $(INC_DIR)/sparsehash/sparse_hash_map
 	+. ./source_me.sh && $(CXX) $(CXXFLAGS) -c $(XG_DIR)/src/xg.cpp -o $(CWD)/$(OBJ_DIR)/xg.o $(LD_INCLUDE_FLAGS) -I$(INC_DIR)/dynamic && ar rs $(CWD)/$(LIB_DIR)/libxg.a $(CWD)/$(OBJ_DIR)/xg.o $(CWD)/$(CPP_DIR)/vg.pb.o && cp $(XG_DIR)/src/*.hpp $(CWD)/$(INC_DIR)/
 
-$(LIB_DIR)/libvcflib.a: $(VCFLIB_DIR)/include/* $(VCFLIB_DIR)/src/* $(VCFLIB_DIR)/intervaltree/* $(VCFLIB_DIR)/lib/*
+$(LIB_DIR)/libvcflib.a: $(VCFLIB_DIR)/src/* $(VCFLIB_DIR)/intervaltree/*
 	+. ./source_me.sh && cd $(VCFLIB_DIR) && $(MAKE) libvcflib.a && cp lib/* $(CWD)/$(LIB_DIR)/ && cp include/* $(CWD)/$(INC_DIR)/ && cp intervaltree/*.h $(CWD)/$(INC_DIR)/ && cp src/*.h* $(CWD)/$(INC_DIR)/
 
 $(LIB_DIR)/libgssw.a: $(GSSW_DIR)/src/gssw.c $(GSSW_DIR)/src/gssw.h

